app/config.py

```diff
@@ -72,6 +90,8 @@ class Development(Config):
     HTTP_SCHEME = "http"
     FRONTEND_HOSTNAME = "localhost:7001"
 
+    REDIS_URL = os.getenv("REDIS_URL", "redis://localhost:6379/0")
```

Comment:
On PaaS we are using DB 1 rather than DB 0. Should we mimic this in development too? Or is there a reason not to?

Reply:
It broadly doesn't matter at all but I'll change it to DB 1 for consistency's sake.